syft/frameworks/torch/mpc/przs.py

```diff
@@ -5,7 +5,7 @@
 import torch
 import syft
 
-RING_SIZE = 2 ** 32
+MAX_LIMIT = 2 ** 32
```

Comment:
This is not really the ring size - is the maximum value for the random value (for the common randomness)

Reply:
Yep...it is. will redo